app/templates/cookies/cookie_settings.html

```diff
@@ -21,12 +21,18 @@
 
 {% block mainContent %}
 
-  {{ dmAlert({
-    "type": "error",
-    "classes": "dm-cookie-settings__error",
+  {{ govukErrorSummary({
     "titleText": "There was a problem saving your settings",
-    "text": "Please select 'Yes' or 'No'.",
-    "attributes": [("id", "dm-cookie-settings-error")]
+    "classes": "dm-cookie-settings__error",
+     "attributes": {
+      "id": "dm-cookie-settings-error"
+    },
+    "errorList": [
+      {
+        "text": "Please select 'Yes' or 'No'",
```

Comment:
According to the [Design System](https://design-system.service.gov.uk/components/radios/#error-messages), this should be something like "Select yes to accept analytics cookies"
```

Reply:
If following Design System, it should be: Select yes if you want to accept analytics cookies. This seems a bit long and could be even simpler, so agree with @domoscargin - Select yes to accept analytics cookies